CASMHMS-5568: EX2500 cabinet doc changes (#2060)
Cray-HPE/docs-csm: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

    ```

1. Collect the following information from the NCN:

>     1. (`ncn-m#`) Determine the component name (xname) of the NCN by referring to the HMN of the system's SHCD file, if it has not been determined yet.

Reviewer: Does this procedure need to be run on a master NCN? Or would it work on any NCN?

Author: I did the prereq stuff for the `Add a Standard Rack Node` procedure. I'm not sure what pre-req stuff is required for the `Add NCN Data` procedure.

Reviewer: I made two PRs with the remaining linting suggestions I have, along with the addition of the prereq section for that page:
https://github.com/Cray-HPE/docs-csm/pull/2075
https://github.com/Cray-HPE/docs-csm/pull/2076

I think if you pull those in, my concerns are all addressed.

        |   |                | SHCD Column to reference | Description
        | - | -------------- | ------------------------ | -----------
        | X | Cabinet number | Source Rack (K20)        | The Cabinet or rack number containing the Management NCN.

>         | C | Chassis number |                          | For air-cooled nodes located within a standard rack, the chassis is `0`. If the air-cooled node is located within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.

Reviewer: ```suggestion
        | C | Chassis number |                          | For air-cooled nodes within a standard rack, the chassis is `0`. If the air-cooled node is within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.
```

>         | C | Chassis number |                          | For air-cooled nodes located within a standard rack the chassis is `0`. If the air-cooled node node is located within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.

Reviewer: ```suggestion
        | C | Chassis number |                          | For air-cooled nodes located within a standard rack the chassis is `0`. If the air-cooled node is located within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.
```

        export XNAME=x3000c0s4b0n0
        ```

>     1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Determine the NCN BMC xname by removing the trailing `n0` from the NCN node xname:

Reviewer 2: ```suggestion
    1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Determine the NCN BMC xname by removing the trailing `n0` from the NCN xname:
```

        |   |                    | SHCD Column to reference   | Description
        | - | ------------------ | -------------------------- | ----
        | X | Cabinet number     | Destination Rack (R20)     | The Cabinet or rack number containing the management NCN.

>         | C | Chassis number     | | For air-cooled management switch located within a standard rack the chassis is `0`. If the air-cooled management switch is located within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.

Reviewer: ```suggestion
        | C | Chassis number     | | For air-cooled management switches within standard racks, the chassis is `0`. If the air-cooled management switch is within an air-cooled chassis in an EX2500 cabinet, then this should be `4`.
```

        | W | Slot/Rack U        | Destination Location (S20) | The Slot/Rack U that the management switch occupies.
        | J | Switch port number | Destination Port (U20)     | The switch port on the switch that the NCN BMC is cabled to.

        ```bash
        export MGMT_SWITCH_CONNECTOR=x3000c0w14j48
        ```

>     1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Determine the xname of the management switch by removing the trailing `jJ` from the `MgmtSwitchConnector` xname:

Reviewer 2: ```suggestion
    1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Determine the xname of the management switch by removing the trailing `jJ` from the `MgmtSwitchConnector` xname.
```

        * Alternatively, view the MAC address table on the management switch that the BMC is cabled to.

>             1. (`ncn-m#`) Determine the alias of the management switch that the BMC is connected to:

Reviewer 2: ```suggestion
            1. (`ncn-m#`) Determine the alias of the management switch that is connected to the BMC.
```

                sw-leaf-bmc-001
                ```

>             1. (`ncn-m#`) SSH into the management switch that the BMC is connected to:

Reviewer 2: ```suggestion
            1. (`ncn-m#`) SSH into the management switch that is connected to the BMC.
```

                ```bash
                ssh admin@sw-leaf-bmc-001.hmn
                ```

            1. (`ncn-m#`) Locate the switch port that the BMC is connected to and record its MAC address.

>                In the commands below, change the value of `1/1/39` to match the BMC switch port number (the BMC Switch port number is the `J` value in the in the `MgmtSwitchConnector` xname `xXwWjJ`).

Reviewer: ```suggestion
               In the commands below, change the value of `1/1/39` to match the BMC switch port number (the BMC switch port number is the `J` value in the in the `MgmtSwitchConnector` xname `xXwWjJ`).
```

                a4:bf:01:65:68:54    4        dynamic                   1/1/48
                ```

>     1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Set the `BMC_MAC` environment variable to the BMC MAC address:

Reviewer 2: ```suggestion
    1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Set the `BMC_MAC` environment variable to the BMC MAC address.
```

        ```bash
        export BMC_MAC=a4:bf:01:65:68:54
        ```

>     1. **Skip if adding `ncn-m001`:** Determine the current IP address of the NCN BMC:

Reviewer 2: ```suggestion
    1. **Skip if adding `ncn-m001`:** Determine the current IP address of the NCN BMC.
```

>         1. (`ncn-m#`) Query Kea for the BMC MAC address to determine its current IP address:

Reviewer 2: ```suggestion
        1. (`ncn-m#`) Query Kea for the BMC MAC address to determine its current IP address.
```

            1. Verify that the BMC is powered up and has an active connection to the network.
            1. Verify that the BMC is set to DHCP instead of a static IP address.

>         1. (`ncn-m#`) Ping the BMC to see if it is reachable:

Reviewer 2: ```suggestion
        1. (`ncn-m#`) Ping the BMC to see if it is reachable.
```

    sleep 60
    ```

> 1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Verify that the BMC is reachable at the expected IP address:

Reviewer: ```suggestion
1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Verify that the BMC is reachable at the expected IP address.
```

    ```bash
    ping $NODE-mgmt
    ```

>     Wait 5 minutes for Kea and the HSM to sync. If `ping` continues to fail, re-run the previous step to restart the BMC.

Reviewer: ```suggestion
    Wait five minutes for Kea and the HSM to sync. If `ping` continues to fail, then re-run the previous step to restart the BMC.
```

    deployment "cray-reds" successfully rolled out
    ```

> 1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Wait for the NCN BMC to get discovered by HSM:

Reviewer: ```suggestion
1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Wait for the NCN BMC to get discovered by HSM.
```

>     > If the BMC of `ncn-m001` is connected to the site network, then we will be unable to discover the BMC, because it is not connected via the HMN network.

Reviewer: ```suggestion
    > If the BMC of `ncn-m001` is connected to the site network, then the BMC will not be discovered, because it is not connected via the HMN network.
```

        Address: 10.254.1.13
        ```

>     * (`ncn-m#`) Verify that the BMC is reachable at the expected IP address.

Reviewer: ```suggestion
    1. (`ncn-m#`) Verify that the BMC is reachable at the expected IP address.
```

        ```bash
        ping $NODE-mgmt
        ```

>     * (`ncn-m#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.

Reviewer: ```suggestion
    1. (`ncn-m#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.
```
